cs251tk/toolkit/__main__.py

```diff
@@ -139,18 +139,30 @@ def main():
     if assignments:
         available_specs = set(assignments)
 
+        if ci:
+            ignored_specs = open('.cs251tkignore').read().split('\n')
+            logging.debug("Ignored specs: {}".format(ignored_specs))
+
+            for spec_to_use in assignments:
+                if spec_to_use in ignored_specs:
+                    available_specs.remove(spec_to_use)
+                    continue
+
         for spec_to_use in assignments:
             try:
                 check_dependencies(specs[spec_to_use])
             except KeyError:
-                print('Spec {} does not exist'.format(spec_to_use), file=sys.stderr)
+                if spec_to_use != 'lab0':
```

Comment:
This smells like test debugging code?

Reply:
This is used to prevent the lab0 directory that some students have from causing an unnecessary message to be printed

Reply:
Can you put in a comment, please?